src/ApiManagement/ApiManagement/Az.ApiManagement.psd1

```diff
@@ -8,197 +8,247 @@
 
 @{
 
-# Script module or binary module file associated with this manifest.
-# RootModule = ''
+    # Script module or binary module file associated with this manifest.
```

Comment:
@solankisamir any reason this file was reformatted? When we version modules during the release process, we run the `Update-ModuleManifest` cmdlet to update specific properties of the `.psd1` files, which will reformat the contents of the file in a specific way, so my worry is that all of the changes made in this file (outside of the additions of the new cmdlets) will be overridden during the release.

Reply:
I added `new` cmdlets and updated `FormatsToProcess`. Apart from that the change, was only to make the file `readable`.

---
In reply to: [283432688](https://github.com/Azure/azure-powershell/pull/9167#discussion_r283432688) [](ancestors = 283432688)